spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -769,5 +769,50 @@
         end
       end
     end
+
+    describe "gubernatorial elections fund" do
+      context "not mfj" do 
+        let(:intake) { 
+          create(:state_file_nj_intake)
+        }
+        it "does not fill the field when the answer is no" do 
+          expect(xml.at("Body PrimGubernElectFund")).to eq(nil)
+          expect(xml.at("Body SpouCuPartPrimGubernElectFund")).to eq(nil)
+        end
+
+        it "checks yes when the answer is yes" do
+          intake.primary_contribution_gubernatorial_elections = :yes
+          expect(xml.at("Body PrimGubernElectFund").text).to eq('X')
```

Comment:
[dust] Would love a quick line in here to also assert spouse is nil in this case too